chemicaltoolbox/xchem-deep/xchem_deep_score.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="xchem_pose_scoring" name="XChem pose scoring" version="0.1.0">
+    <description>- perform pose scoring using deep learning</description>
+
+    <requirements>
+        <!--requirement type="package" version="3.0.0">openbabel</requirement-->
+        <!--requirement type="package" version="3.7">python</requirement-->
+        <!-- many other requirements are needed -->
+        <container type="docker">informaticsmatters/deep-app-ubuntu-1604:latest</container>
```

Comment:
this should be a tagged version

Reply:
Don't have one yet. But we will at some stage soon.